UI/UISearchBar.cs

```diff
@@ -107,8 +107,17 @@ private void HandleTextInput()
 				prev = prev.Remove(cursorPosition);
 
 			string newString = Main.GetInputText(prev);
-			if ((Main.keyState.IsKeyDown(Keys.LeftControl) || Main.keyState.IsKeyDown(Keys.RightControl)) && KeyTyped(Keys.Back))
-				DeleteWord(ref newString);
+
+
+			if (Main.keyState.IsKeyDown(Keys.LeftControl) || Main.keyState.IsKeyDown(Keys.RightControl))
+			{
+				if (KeyTyped(Keys.Back))
+					DeleteWord(ref newString);
+				else if (KeyTyped(Keys.Left))
+					HandleLeft();
+				else if (KeyTyped(Keys.Right))
+					HandleRight();
+			}
```

Comment:
Does this behave correctly when Ctrl + Left/Right and some letter is pressed at the same time?

Reply:
Works as you expect for me. You can't input anything while holding down CTRL + Both arrow keys.

Reply:
Ah right. Ctrl + letter key doesn't count as a letter